cmd/in-toto/record.go

```diff
@@ -138,6 +138,14 @@ command is executed. Symlinks are followed.`,
 are stored in the resulting link metadata after the
 command is executed. Symlinks are followed.`,
 	)
+
+	recordCmd.Flags().BoolVarP(
+		&lineNormalization,
+		"line-normalization",
+		"o",
+		false,
+		`Enable line normalization.`,
```

Comment:
Ideally, the help message should talk a bit more about what enabling this flag does, perhaps with a quick example, and when to use it.